internal/services/containerapps/container_app_environment_storage_resource.go

```diff
@@ -95,6 +98,14 @@ func (r ContainerAppEnvironmentStorageResource) Arguments() map[string]*pluginsd
 			}, false),
 			Description: "The access mode to connect this storage to the Container App. Possible values include `ReadOnly` and `ReadWrite`.",
 		},
+
+		"nfs_server": {
```

Comment:
I was preparing the change log entry for this and realized it isn't clear what is being asked for here, is this a nfs server name url hostname or?

should we make this
```suggestion
		"nfs_server_host_name": {
```

or can it be many things?

could we also update the documentation as to the expected value?

Reply:
@katbyte , this should be an url for the server. I'll change to `nfs_server_url` to make it more clear.